docs/atomic-install.1.md

```diff
@@ -60,6 +61,11 @@ If --display is not specified the install command will execute.
  Use this name for creating installed content for the container.
  NAME will default to the IMAGENAME if it is not specified.
 
+**--remote=REMOTE**
+Specify a REMOTE folder, which should be an existing, expanded
+container. The existing container's rootfs will be used as the
+new system container's rootfs. It is only used by --system.
```

Comment:
I actually tested it and it works (after I manually changed the rootfs in the configuration file, the image I've used had a template conf file), so the change in the documentation is enough.